index.html

```diff
@@ -1,11 +1,99 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Challenge 1</title>
+  <title>Gatitos Molones/Challenge 1</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Challenge 1</h1>
+  <h1>Mi Web</h1>
+  <hr>
+  <div class="Logo-español">
+    <a href="https://translate.google.com/?hl=es" target="_blank"><img width="90px" height="60x"
+        src="https://media.istockphoto.com/id/637194620/es/foto/bandera-de-espa%C3%B1a-icono-redondo-con-marco-met%C3%A1lico.jpg?s=170667a&w=0&k=20&c=ZGSikUNfKNFKHV4QJrdiPsKbJH2xcMyyN49OYANJMFA="
+        alt="Link al traductor"></a>
+  </div>
+  <div class="caja-1-container">
+    <img width="700" height="400"
+      src="https://img.freepik.com/fotos-premium/gato-gafas-sol-sienta-playa_900958-3265.jpg" alt="gato con estilo">
+    <div class="caja-info">
+      <div class="galeria">
+        <a class="link-galeria" href="https://www.catit.com/es/mi-gato/galerias-de-gatos/" target="_blank">
+          <strong>Galería: adopta un gatito</strong></a>
+      </div>
+      <div class="boton-1">
+        <a class="boton-fake" href="https://www.feelcats.com/razas-gatos/" target="_blank">Clasificación</a>
+        <div class="fecha">
+          <p>Agosto 23, 2023</p>
+        </div>
+      </div>
+      <h2>¿Por qué los gatos deberían ser los mejores amigos del hombre?</h2>
+      <p class="info-1">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Explicabo, hic fugiat nihil asperiores
+        debitis incidunt doloribus est quo voluptate possimus tempora eos accusantium alias fuga autem eaque mollitia!
+        Iusto!</p>
+      <div class="by">
+        <p class="mini-cajita"> by <span style="color: black;"><strong> Vick & Angie</strong></span></p>
```

Comment:
Procuremos evitar inline-styles a no ser que sea estrictamente necesario
```suggestion
        <p class="mini-cajita"> by <span><strong> Vick & Angie</strong></span></p>
```